xcube/core/gen2/config.py

```diff
@@ -142,6 +142,7 @@ def __init__(self,
                  tile_size: Union[int, Tuple[int, int]] = None,
                  time_range: Tuple[str, Optional[str]] = None,
                  time_period: str = None,
+                 temporal_resampling: str = None,
```

Comment:
See above.

Reply:
Closing to keep the conversation in one comment.